docs/README.md

```diff
@@ -1998,18 +2017,13 @@ Booster.configure('prod', (config: BoosterConfig): void => {
 })
 ```
 
-It is also possible to place an environment configuration in a separated file, for instance:
+It is also possible to place an environment configuration in a separated file. Let's say that a developer called "Pepe" created its own configuration file `src/config/pepe.ts`. The content would be the following:
```

Comment:
For some reason (don't ask why) some people link the Pepe name towards right-wing fascist movements. I'd change this to `John Doe` or something

Reply:
Oh God! I'll change it

Reply:
Done b8f42e1